File: get_feature_279.js
const sqlite3 = require('sqlite3').verbose();
const db = new sqlite3.Database('./assistant.db');

db.get('SELECT id, category, name, description, steps, passes, in_progress, dependencies FROM features WHERE id = 279', (err, row) => {
  if (err) {
    console.error(err);
    process.exit(1);
  }

  if (row) {
    console.log('Feature #279:');
    console.log('Category:', row.category);
    console.log('Name:', row.name);
    console.log('Description:', row.description);
    console.log('Steps:', row.steps);
    console.log('Passes:', row.passes);
    console.log('In Progress:', row.in_progress);
    console.log('Dependencies:', row.dependencies);
  } else {
    console.log('Feature #279 not found');
  }

  db.close();
});

File: testing-docs/get_feature_279.py
#!/usr/bin/env python3
import sqlite3
import json

db = sqlite3.connect('./assistant.db')
cursor = db.cursor()

cursor.execute('SELECT id, category, name, description, steps, passes, in_progress, dependencies FROM features WHERE id = 279')
row = cursor.fetchone()

if row:
    print(f"Feature #{row[0]}:")
    print(f"Category: {row[1]}")
    print(f"Name: {row[2]}")
    print(f"Description: {row[3]}")
    print(f"Steps: {row[4]}")
    print(f"Passes: {row[5]}")
    print(f"In Progress: {row[6]}")
    print(f"Dependencies: {row[7]}")
else:
    print("Feature #279 not found")

db.close()
